test/inductor/test_cudacodecache.py

```diff
@@ -37,6 +38,7 @@
 """
 
 
+@unittest.skipIf(IS_S390X, "No CUDA on S390X")
```

Comment:
Is it possible to use ``from torch.testing._internal.inductor_utils import HAS_CPU
`` here and add S390X case to it?

Reply:
Sorry typo, I meant `HAS_GPU`. Your can check that in `torch/testing/_internal/inductor_utils.py` line 64